lib/caew_corporate_importer/push_contentful.rb

```
@@ -60,10 +62,13 @@ def content_types
     end
 
     def cf_epiblob(epipage)
+      title = "#{epipage.json['id']}: #{epipage.json['mainTitle']}"
+      json_text = epipage.json.to_json[..EPISERVER_MAX_TEXT_LIMIT]
```

If we truncate JSON like this by chopping the json, we risk no longer having valid JSON right? 

We're trying to be good citizens by keeping a copy of this data though we don't have a clear idea as to its purpose. It might be a useful reference, it might help content users reference what was or it might never be used. If it _is_ used the data can be valid or in this case invalid JSON, without a way for a person to find the complete extent of the original document to see how much is missing. 

Arguably some legacy data is better than none at all. I think there are 3 options:

1. Set an expectation to the user that this is no longer valid JSON (maybe add a new line at the start?)
2. Don't store any legacy data if a truncation occurs, the person will have to find another backup
3. Fail loudly during the migration and ask a user to split the chunk into multiple episerverBlobs that are each valid JSON

Number 2 is probably easiest.

---

I'm tempted by 'fail loudly and replace data: uris with empty strings in a really simple way'